examples/gcoap/Makefile.slip

```diff
@@ -0,0 +1,84 @@
+# Border router Makefile for SLIP based networking
+# Assumes use of SAMR21 board
+
+# name of your application
+APPLICATION = gcoap
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= samr21-xpro
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+BOARD_INSUFFICIENT_MEMORY := chronos msb-430 msb-430h nucleo-f030 nucleo-f334 \
+                             stm32f0discovery telosb weio wsn430-v1_3b wsn430-v1_4 \
+                             z1
+
+# Must read nordic_softdevice_ble package before nanocoap package. However,
+# can't read it explicitly here because it is read later, as a dependency for
+# the nrf52dk board.
+BOARD_BLACKLIST := nrf52dk
+
+# Redefine port, for example use 61616 for RFC 6282 UDP compression.
+#GCOAP_PORT = 5683
+#CFLAGS += -DGCOAP_PORT=$(GCOAP_PORT)
+
+# Redefine request token length, max 8.
+#GCOAP_TOKENLEN = 2
+#CFLAGS += -DGCOAP_TOKENLEN=$(GCOAP_TOKENLEN)
```

Comment:
Could you maybe add some comment about than (like "uncomment if needed" like in the [`gnrc_minimal`](https://github.com/RIOT-OS/RIOT/blob/master/examples/gnrc_minimal/Makefile#L12) example)?